daliuge-engine/dlg/deploy/helm_client.py

```diff
@@ -383,8 +383,8 @@ def submit_and_monitor_pgt(self):
         """
         Combines submission and monitoring steps of a pgt.
         """
-        session_id = self.submit_pgt()
-        monitoring_thread = self._monitor(session_id)
+        self.submit_pgt()
```

Comment:
`self.submit_pgt` was never returning `session_id` (and from what I can tell, that would be a bit of work to derive). Hence, we've always been passing `None` as a parameter all this time. It is cleaner to just remove this.